Application/EdFi.Ods.WebApi/appsettings.development.json

```diff
@@ -0,0 +1,28 @@
+﻿{
```

Comment:
This development settings file shouldn't be added here explicitly -- it gets created via a Powershell script.

Reply:
Removed it